data/ui/notes.ui

```diff
@@ -65,12 +67,13 @@
                     <property name="vexpand">true</property>
                     <property name="visible">false</property>
                     <property name="icon-name">document-edit-symbolic</property>
-                    <property name="title" translatable="true">No notes</property>
+                    <property name="title" translatable="true">No Saved Notes</property>
                     <property name="description" translatable="true">After you create notes, they will appear here</property>
                     <child>
                       <object class="GtkButton">
                         <property name="halign">center</property>
-                        <property name="label" translatable="true">Create a new note</property>
+                        <property name="label" translatable="true">_New Note</property>
```

Comment:
why is this better??

"Create a new note" insists that "new" is used as an action (verb) and not describing a note (new/old note) as an adjective
```suggestion
                        <property name="label" translatable="true">Create a new _Note</property>
```

Reply:
How about "Create _New Note"?

Reply:
According to https://developer.gnome.org/hig/patterns/controls/buttons.html

-     Button labels should use imperative verbs, using header capitalization. For example, Save or Update.

-     Button labels should be short, in order to keep the button width narrow. Consider how labels will change width when localized.

Reply:
I changed it to "Create _New Note".

Reply:
please revert it. I realised that "New Note" is better and sorry for wasting your time.

Reply:
No problem. ^^ I'll revert really soon.

Reply:
Done!